functions/typed/Typed.Tests/TypedGreetingTest.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Functions.Testing;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+using Greeting;
+using System.Text;
+
+namespace Typed.Tests;
+
+public class TypedGreetingTest : FunctionTestBase<TypedFunction>
```

Comment:
I'd rename this to GreetingTest - renaming the file too - otherwise it sounds like there's a class or namespace called "TypedGreeting".

(I think the namespace and project name is fine though.)